NuGet.config

```diff
@@ -19,6 +19,7 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="orleans-nightly" value="https://orleans.pkgs.visualstudio.com/orleans-public/_packaging/orleans-nightly/nuget/v3/index.json" />
     <add key="dotnet9-transport" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet9-transport/nuget/v3/index.json" />
+    <add key="dotnet8-early" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/dotnet8-early-access/nuget/v3/index.json" />
```

Comment:
Remove I think.

Reply:
would remove in final PR yes...needed just during WIP

Reply:
What's it needed for? The 2 new packages are on nuget.org. We can just get the mirrored internally. https://github.com/dotnet/arcade/blob/main/Documentation/MirroringPackages.md

Reply:
Hmm, at least for me locally I was failing on some 8.0.5 packages and it wasn't finding the 8.0.0 Aspire ones

Reply:
> was failing on some 8.0.5 packages 

that must have been a point in time thing as 8.0.5 was just released yesterday.